src/components/BackButton.js

```diff
@@ -16,6 +18,17 @@ class BackButton extends Component {
 		};
 	}
 
+	label() {
+		var label = "";
+		if (this.props.label == null) {
+			label = "Back";
+		}
+		if (this.props.label == "cancel") {
+			label = "Cancel";
+		}
+		return label;
+	}
```

Comment:
Simpler way that allows for multiple different labels: `return this.props.label || "Back";` This allows for any non-hard coded label why also setting a default.